src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/transport/action/updateindexmetadata/TransportUpdateManagedIndexMetaDataAction.kt

```diff
@@ -65,6 +67,14 @@ class TransportUpdateManagedIndexMetaDataAction : TransportMasterNodeAction<Upda
         this.indexStateManagementHistory = indexStateManagementHistory
     }
 
+    companion object {
+        val reader = Writeable.Reader {
```

Comment:
Am I missing something or is this reader not being use?

Reply:
Forgot to remove, thanks, cleaned up.